cocotb/_build_libs.py

```diff
@@ -38,6 +38,28 @@ def run(self):
 
         super().run()
 
+    # On osx need extra extra rpath with install_name_tool to find libraries
+    if sys.platform == "darwin":
+        def build_extension(self, ext):
+            super().build_extension(ext)
+
+            subprocess.run(
+                [
+                    "install_name_tool",
+                    "-add_rpath",
+                    "@loader_path",
+                    os.path.join(self.build_lib, ext._file_name),
+                ]
+            )
+            subprocess.run(
+                [
+                    "install_name_tool",
+                    "-add_rpath",
+                    "@loader_path/libs",
+                    os.path.join(self.build_lib, ext._file_name),
+                ]
+            )
```

Comment:
Needs a `check=True` argument.
```suggestion
            subprocess.run([
                "install_name_tool",
                "-add_rpath",
                "@loader_path",
                os.path.join(self.build_lib, ext._file_name),
            ], check=True)
            subprocess.run([
                "install_name_tool",
                "-add_rpath",
                "@loader_path/libs",
                os.path.join(self.build_lib, ext._file_name),
            ], check=True)
```